src/scripts/browserdeviceprofile.js

```diff
@@ -355,6 +355,31 @@ define(['browser'], function (browser) {
         // Not sure how to test for this
         var supportsMp2VideoAudio = browser.edgeUwp || browser.tizen || browser.orsay || browser.web0s;
 
+        // Get Tizen version & support
+        var getTizenVersion = self.tizen && self.tizen.systeminfo ? 
+            parseFloat(tizen.systeminfo.getCapability('http://tizen.org/feature/platform.version')) : null;
```

Comment:
_tbh, I see this code section first time_ :relaxed:

I know that it was before, but it will work in app only.
I was told that we should support browser users as possible. Personally, I see no reason to use browser when you have app, but why not if we can.

I recently added getting Tizen version.
https://github.com/jellyfin/jellyfin-web/blob/d0c2c1c8e0fe605f5ddc416cdbff81056faa2f5b/src/scripts/browser.js#L275-L276
We could use that. `browser.tizenVersion` is integer (enough for now).

Also, there are lint errors:
```
  359:68  error    Trailing spaces not allowed                             no-trailing-spaces
  364:21  warning  'version' is already defined                            no-redeclare
  365:17  error    Expected space(s) after "switch"                        keyword-spacing
```

Reply:
If we release an app compatible with all tizen devices I would say we can ignore tizen browser support from that point. Depending on how old tizen can be I would even say just the latest versions are fine.

Reply:
I would say browser support is required until we can apply for an official release though, and I have no idea what the timeline is for that.

Reply:
> If we release an app compatible with all tizen devices I would say we can ignore tizen browser support from that point.

Not "if" but "when" :wink: 
iirc, this is a second place where Tizen API is used, and we could remove both.
Btw, I already checked what I suggest.

>  Depending on how old tizen can be I would even say just the latest versions are fine.

Currently, we already support Tizen 3+ (with `query-string` being "babeled"). There was issue from a guy who owned 2.3 or 2.4, but he didn't publish results or just had no time, because building a Tizen app is a pain.